packages/nodes-base/nodes/TheHive/TheHiveTrigger.node.ts

```diff
@@ -148,29 +146,35 @@ export class TheHiveTrigger implements INodeType {
 			return {};
 		}
 
+		// While TheHive uses "deleted" for the other objects, a task is not deleted but is updated with the status "Cancel"
+		if (
+			bodyData.operation === 'update' &&
+			bodyData.objectType === 'case_task' &&
+			(bodyData.details as IDataObject)?.status === 'Cancel'
+		) {
+			bodyData.operation = 'delete';
```

Comment:
I did add condition to check if trigger listening for task delete, and only then replace update>cancel with delete, otherwise it will behave like before